os/kernel/binary_manager/binary_manager.c

```diff
@@ -197,4 +205,20 @@ int binary_manager(int argc, char *argv[])
 	}
 
 	return 0;
+errout_with_nobinary:
+	while (1) {
+		lldbg("=============== !!ERROR!! ============== \n");
+#ifdef CONFIG_APP_BINARY_SEPARATION
+		if (!is_found_ubin) {
+			lldbg("ERROR!! Not found valid user binaries.\n");
+			lldbg("Please check user binary and smartfs configurations.\n");
+		}
+#endif
+		if (!is_found_kbin) {
+			lldbg("ERROR!! Not found kernel partitions.\n");
```

Comment:
`binary_manager` is also running on kernel, so I think this message can make confusion.
Please modify the message with more detail.